modin/core/dataframe/pandas/utils.py

```diff
@@ -32,13 +32,13 @@ def concatenate(dfs):
     pandas.DataFrame
         A pandas DataFrame.
     """
-    categoricals_columns = set.intersection(
-        *[set(df.select_dtypes("category").columns.tolist()) for df in dfs]
-    )
-
-    for col in categoricals_columns:
-        uc = union_categoricals([df[col] for df in dfs])
+    for i in range(len(dfs[0].columns)):
```

Comment:
Let's modify function's docstring so it says that the `dfs` columns have to be equal and add some assertion verifying that
```suggestion
    for df in dfs:
        assert df.columns.equals(dfs[0].columns)
    for i in range(len(dfs[0].columns)):
```

Reply:
> Let's modify function's docstring so it says that the dfs columns have to be equal 

Done.

Reply:
> Let's modify function's docstring so it says that the `dfs` columns have to be equal 

Done.